packages/emotion-server/test/auto-label/__snapshots__/auto-label.test.js.snap

```diff
@@ -2,15 +2,7 @@
 
 exports[`extractCritical returns static css 1`] = `
 Object {
-  "css": "@font-face {
```

Comment:
I might need some help with this, tried for a few hours still no luck to get `injectGlobal` work